examples/simulated/plot_mean_median_comparison.py

```diff
@@ -0,0 +1,148 @@
+"""
+===============================================================================
+Mean and median comparison
+===============================================================================
+
+A comparison between Euclidean and Riemannian means [1]_, and Euclidean and
+Riemannian geometric medians [2]_, on low-dimensional synthetic datasets.
+
+"""
+# Authors: Quentin Barthélemy
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.datasets import make_blobs
+from pyriemann.datasets import make_outliers
+from pyriemann.utils.mean import (
+    mean_euclid, mean_riemann, median_euclid, median_riemann
+)
```

Comment:
we should be able to do:

```suggestion
from pyriemann.utils import (
    mean_euclid, mean_riemann, median_euclid, median_riemann
)
```

importing median from mean is weird and it would reduce the nesting of the code.